bridge/Bridge.cpp

```diff
@@ -153,7 +153,9 @@ void Bridge::initialize(std::shared_ptr<transport::EndpointFactory> endpointFact
     }
 
     startWorkerThreads();
-    backgroundWorker = std::make_unique<jobmanager::WorkerThread>(*_backgroundJobQueue, "MMWorker");
+    // Disabling yield because we don't want to yield jobs that holds mixer and MixerManager locks, otherwise it can create deadlocks
+    const bool yieldEnabled = false;
```

Comment:
Yielding jobs that holds the configuration lock, can potential create deadlocks if another job that needs the lock is picked